File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.cs
using System; 
public class Solution {
    public long solution(long n) {
        string s = n.ToString();
        char[] arr = s.ToCharArray();
        Array.Sort(arr, (a, b) => b.CompareTo(a));
        string sort = new string(arr);
        long answer = long.Parse(sort);
        
        return answer;
    }
}

File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

long long solution(long long n) {
    string s = to_string(n);
    vector<char> v;
    for(char c : s)
    {
        v.push_back(c);
    }
    
    sort(v.begin(), v.end(), greater<char>());
    string result(v.begin(), v.end());
    long long answer = stoll(result);
    return answer;
}